__tests__/fluentBit.spec.ts

```diff
@@ -129,6 +109,97 @@ describe('fluentBit', () => {
     `);
   });
 
+  it('Parses global @includes in configuration', async () => {
+    const filePath = './__fixtures__/includes/withIncludes.conf';
+    const rawConfig = readFileSync(filePath, { encoding: 'utf-8' });
+    const config = new FluentBitSchema(rawConfig, filePath);
+    expect(config.AST).toMatchInlineSnapshot(`
```

Comment:
So this is testing your AST matches this after trying to parse the input file. I'm not a huge fan of big string lists like this, it would have been nicer to keep the expected output alongside the input and just iterate over all of them we have rather than having to make it so explicit in code. Is that doable, i.e. input + output for comparison?

Reply:
Resolved.

Reply:
sweet. closing